about.html

```diff
@@ -0,0 +1,119 @@
+          emergence of one of the greatest players of all time, Rafael Nadal. Nadal has 
+          won a record 13 French Open titles, and his dominance on the clay courts of 
+          Roland Garros is unmatched. He has won the tournament almost every year since 
+          his first win in 2005, except for a few years when he was either injured or 
+          unable to compete. Nadal’s success at the tournament has solidified his place 
+          in tennis history and has made him a household name around the world. His rivalry 
+          with Roger Federer and Novak Djokovic has also added to the tournament’s appeal, 
+          making it a highly anticipated event every year.
+      </p>
+      <p class="lato about-info">Please contact us per email for any further questions about Roland Garros Tournament!</p>
+      <h5 class="lato">info@rolandgarros.com</h5>
+    </section>
+    <section class="about-details" id="about-logo">
+      <h2 class="lato">Roland Garros Tournament <span></span></h2>
+      <p class="lato">Our logo it reminds of the fact that Roland Garros is often named the birthplace of 
+        the ground court game. It is paired with white to resemble the colors of a stadium. 
+        The green on the logo symbolizes the trees and lawns seen around the courts.</p>
+      <img src="./assets/images/logo-rg.svg" alt="Roland Garros">
+    </section>
+    <section class="about-details">
+        <h2 class="lato">See the last winners in this tournament <span></span></h2>
+        <p class="lato">Take a look at the last two winners which took place in 2021 and 2022.</p>
+        <div class="past-events">
+          <figure>
+            <img src="./assets/images/about-01.png" alt="Roland Garros 2021">
+            <figcaption class="lato"><span class="caption">2021</span> <br>Winner: Novak Djokovic
+            </figcaption>
+          </figure>
+          <figure>
+            <img src="./assets/images/about-02.png" alt="Roland Garros 2022">
+            <figcaption class="lato"><span class="caption">2022</span> <br>Winner: Rafael Nadal</figcaption>
+          </figure>
+        </div>
+    </section>
+    <section id="partners">
+      <h2 class="lato">Partners <span></span></h2>
+      <ul>
+        <li><img src="./assets/images/BNP-Paribas.png" alt="BNP-Paribas"></li>
+        <li><img src="./assets/images/Lacoste.png" alt="LaCoste"></li>
+        <li><img src="./assets/images/Renault.png" alt="Renault"></li>
+        <li><img src="./assets/images/Rolex.png" alt="Rolex"></li>
+        <li><img src="./assets/images/Wilson.png" alt="Wilson"></li>
+      </ul>
+    </section>
+    <footer class="lato">
+      <img src="./assets/images/logo-rg.svg" class="logo" alt="Roland Garros">
+      <p>©2023 - all rights reserved. Fédération Française de Tennis <br>
+        All content on the website, unless otherwise specified, is available under a Creative Commons Attribution</p>
+    </footer>
```

Comment:
- In the Desktop Size, for the footer :+1: 
About Page: It has a dark background.
Home Page: It has a white background.


Kindly Fix that, by making   the footer for about section dark as required in the project :+1: 

## Your design 
![image](https://user-images.githubusercontent.com/95297251/235261999-4943d931-3da6-43a5-b283-a909861e5bc5.png)

## design provided 

![image](https://user-images.githubusercontent.com/95297251/235261918-eb24c78b-029d-47fc-91d0-77df22da3031.png)

Reply:
![issue-01](https://user-images.githubusercontent.com/86327867/235285474-9a4a2b51-b3fc-4fad-9139-0bae08cf1fec.jpg)

**Fixed** the footer (Desktop size) in about.html.